### PR TITLE
Docs auto-gen improvements

### PR DESCRIPTION
**Pull request by markbackman**
#### Please describe the changes in your PR. If it is addressing an issue, please reference that as well.

Fixes for content formatting and missing packages.

There was one fix for Anthropic due to a type issue with Python 3.12.

Also, added a README and a test script that does a clean install in a venv, so you can confirm that package installs are working correctly.

### CODE REVIEW

File: src/pipecat/services/anthropic.py [hunk 1]
```diff
@@ -328,7 +327,7 @@ def __init__(
         tools: list[dict] | None = None,
         tool_choice: dict | None = None,
         *,
-        system: str | NotGiven = NOT_GIVEN,
+        system: Union[str, NotGiven] = NOT_GIVEN,
```
Comment (markbackman) [inline src/pipecat/services/anthropic.py:330]: Had to change this to comply with python 3.12.

File: pyproject.toml [hunk 2]
```diff
@@ -66,11 +66,11 @@ openpipe = [ "openpipe~=4.38.0" ]
 playht = [ "pyht~=0.1.8", "websockets~=13.1" ]
 riva = [ "nvidia-riva-client~=2.17.0" ]
 silero = [ "onnxruntime~=1.20.1" ]
+simli = [ "simli-ai~=0.1.7"]
```
Comment (markbackman) [inline pyproject.toml:69]: Just keeping things tidy.

File: .readthedocs.yaml [hunk 3]
```diff
@@ -4,12 +4,70 @@ build:
...
+        - openpipe
+        - silero
+        - simli
+        - soundfile
+        - websocket
+        - whisper
+
+formats:
+  - pdf
+  - epub
```
Comment (aconchillo) [inline .readthedocs.yaml:62]: do we need pdf and epub?

**Review by aconchillo — APPROVED** [review]

**Comment by aconchillo** [comment]
LGTM! Just added a question.

File: .readthedocs.yaml (hunk 3 above)
Comment (markbackman) [reply .readthedocs.yaml]: Nope. I think I pulled this from example code. Removing.

Comment (markbackman) [reply .readthedocs.yaml]: I think we can remove all `formats:`

**Pull request merged**